.gitmodules

```
@@ -1,3 +1,3 @@
 [submodule "examples"]
 	path = examples
-	url = ../examples.git
+	url = https://github.com/shuttle-hq/shuttle-examples.git
```

How about correcting the name of the now renamed examples repo, and suggesting that users who fork the main repo also fork and clone the examples repo?
```suggestion
	url = ../shuttle-examples.git
```

---

Yeah sounds about right! I'll change it. Sorry it took so long for answering